harness/determined/common/api/experiment.py

```diff
@@ -127,7 +128,10 @@ def create_experiment(
     archived: bool = False,
     activate: bool = True,
     additional_body_fields: Optional[Dict[str, Any]] = None,
-) -> int:
+) -> Tuple[int, Optional[Sequence[bindings.v1LaunchWarning]]]:
```

Comment:
technically, this is an internal function so you can change this API if you'd like to.

Somewhere I have some work to [clean this up](https://github.com/determined-ai/determined/pull/4914).  So hopefully this function just evaporates eventually.  Not sure that affects what you to this function today though.